src/boot/efi/util.h

```diff
@@ -20,7 +20,7 @@ __attribute__((noreturn)) extern void __assert_cl_failure__(void) __attribute__(
 __attribute__((noreturn)) extern void __assert_cl_failure__(void);
 #endif
 /* assert_cl generates a later-stage compile-time assertion when constant folding occurs. */
-#define assert_cl(condition) if (!(condition)) __assert_cl_failure__()
+#define assert_cl(condition) do { if (!(condition)) __assert_cl_failure__(); } while (0)
```

Comment:
Let's use statement expressions for new code macros: `({})`

Reply:
Will do.